pkgs/top-level/perl-packages.nix

```diff
@@ -22483,21 +22483,6 @@ with self; {
     };
   };
 
-  SearchXapian = buildPerlPackage {
```

Comment:
`SearchXapian` is indeed marked as "Old stable release" on https://xapian.org/download, so indeed I guess it deserves to be removed.